v2/discoveryv2/discovery_v2_integration_test.go

```diff
@@ -311,3 +311,17 @@ func TestEnrichments(t *testing.T) {
 
 	assert.Nil(t, deleteErr)
 }
+
+func TestQueryCollectionNotices(t *testing.T) {
```

Comment:
Do we have full disco v2 tests? are we testing with beta instance?

Reply:
We have a full test suite of generated unit tests, and then select integration tests. We are testing with beta.